### PR TITLE
update ghpages to reports.risczero.com

### PR DESCRIPTION
**Pull request by nuke-web3**

### CODE REVIEW

**Comment by morgangibbs87** [comment]
@nuke-web3 @pdg744 - I updated the branch with recent changes from `main` and killed the Benchmarks checks for this PR since they are not needed given the changes. If the rest of the tests pass feel free to land this.

File: website/package.json [hunk 1]
```diff
@@ -47,5 +47,6 @@
       "last 1 safari version"
     ]
   },
-  "description": "RISC Zero - The ZKVM"
+  "description": "RISC Zero - The ZKVM",
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```
Comment (pdg744) [inline website/package.json:51]: is this change intended?

**Review by flaub — APPROVED** [review]

File: website/package.json (hunk 1 above)
Comment (nuke-web3) [reply website/package.json]: no - must have been auto generated following https://github.com/risc0/risc0/tree/main/website#local-development. Is there a reason we want to let that yarn version float?
 (TIL https://github.com/yarnpkg/berry/discussions/5394  can help you set project and/or global versions)

File: web/apps/benchmarks-and-reports/src/app/(benchmarks-and-reports)/benchmarks/_components/charts.tsx [hunk 2]
```diff
@@ -143,7 +143,7 @@ export function Charts() {
       <FooterAscii text="Benchmarks" />
 
       <Script
-        src="https://risc0.github.io/ghpages/dev/bench/data.js"
+        src="https://reports.risczero.com/benchmarks"
         onReady={() => {
```
Comment (nuke-web3) [inline web/apps/benchmarks-and-reports/src/app/(benchmarks-and-reports)/benchmarks/_components/charts.tsx:147]: this likely was a mistake that broke the vercel job? cc @flaub I should have spotted https://github.com/risc0/ghpages/tree/main/dev/bench/data.js and other scripts are not in the new hosted site I can see. I should not have used this incorrect link either. Sorry! 
image

Comment (flaub) [reply web/apps/benchmarks-and-reports/src/app/(benchmarks-and-reports)/benchmarks/_components/charts.tsx:147]: Yep, I'm on it
